packages/react-components/src/components/AppFrame/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+import cx from 'clsx';
+
+import { INavBarProps } from '../../types';
+
+import styles from './NavBar.module.scss';
+
+const baseClass = 'nav-bar';
+
+export const NavBar: React.FC<INavBarProps> = ({
+  className,
+  topNavBarNode,
+  bottomNavBarNode,
+}) => {
+  return (
+    <div role="navigation" className={cx(styles[baseClass], className)}>
```

Comment:
we can use `nav` element as it more semantically correct and there will be no need to add role attr explicitly. 
Also, this component could also extend the navite HTML <nav> props and spread them to allow developers pass more props if needed. I'd suggest using this strategy for other components as well.